cli/flox-rust-sdk/src/providers/catalog.rs

```diff
@@ -1088,12 +1074,30 @@ impl SearchTerm {
     }
 }
 
+// These functions should really be a #[cfg(test)] method on their
+// respective types, but you can't import test features across crates
 pub mod test_helpers {
     use super::*;
     use crate::data::System;
 
-    // This function should really be a #[cfg(test)] method on ResolvedPackageGroup,
-    // but you can't import test features across crates
+    /// Clear mock responses and then load responses from a file into the list
+    /// of mock responses
+    pub fn reset_mocks_from_file(client: &mut Client, relative_path: &str) {
```

Comment:
bs: impl on `Client`
non blocking - we can go with implementors choice here

Reply:
I'd prefer not to. Added to the description:

> Using the `test_helpers` mod rather than `impl Client` so that we don't
> accidentally use it outside tests, which would only panic at runtime.

Does the "bs" mean "blocking suggestion" but also "non-blocking"?